integration/support/helpers.js

```diff
@@ -1,24 +1,29 @@
-const clickLink = async (page, title) => {
-  return expect(page).toClick('a', { text: title })
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const pupExpect = require('expect-puppeteer')
+
+const clickNavigationTag = tagName => async (page, text) => {
+  // XXX: Some buttons/links don't do anything if you click them too quickly,
+  // so for, now, add a small delay
+  await page.waitFor(500)
```

Comment:
Could you raise a ticket for this?

Reply:
sure thing! I'm going to be working on more e2e tests today / next week so I should get the chance to tool around more here.